contracts/TokenConversionManager.sol

```diff
@@ -20,12 +22,16 @@ contract TokenConversionManager is Ownable, ReentrancyGuard {
     uint256 public perTxnMaxAmount;
     uint256 public maxSupply;
 
+    // The fee percentage will be multiplied by 100 and passed during the configurations.
+    uint256 public feePercentage; 
```

Comment:
So the fee will be the same for every token ?

Reply:
Contract instance will be different for every token, so the fee will be different for every token.